src/main/java/de/tum/in/www1/artemis/service/connectors/BambooService.java

```diff
@@ -530,6 +537,95 @@ public Result onBuildCompleted(Participation participation) {
         return result;
     }
 
+    @Override
```

Comment:
Could you add JavaDoc comments here?

Reply:
Not needed as existing in interface JavaDoc.